models/custom-application/src/custom-application-installation/custom-application-installation/generator.ts

```diff
@@ -0,0 +1,24 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import * as CustomApplicationInstallationPermission from '../../custom-application-installation-permission';
+import type { TCustomApplicationInstallation } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomApplicationInstallation>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    createdAt: fake(getOlderDate),
+    updatedAt: fake(getNewerDate),
+    acceptedPermissions: fake(() => [
+      CustomApplicationInstallationPermission.presets.ViewOnlyPermissions(),
+      CustomApplicationInstallationPermission.presets.ManageOnlyPermissions(),
+    ]),
+    projects: fake(() => []), //TODO: add ProjectExtension
+    installInAllProjects: fake((f) => f.datatype.boolean()),
+    owner: fake(() => ({})),
+    ownerId: fake((f) => f.string.uuid()),
+  },
```

Comment:
The `application` field is missing

Reply:
updated [51f6c10](https://github.com/commercetools/test-data/pull/542/commits/51f6c105e35546649253d4765c2a22af04549d31)